doctr/datasets/synthtext.py

```diff
@@ -96,6 +102,10 @@ def __init__(
             if not os.path.exists(os.path.join(tmp_root, img_path[0])):
                 raise FileNotFoundError(f"unable to locate {os.path.join(tmp_root, img_path[0])}")
 
+            # Skip corrupted images
+            if img_path[0].startswith(self.BLACKLIST):
```

Comment:
```
if img_path[0].startswith(self.BLACKLIST):  # pragma: no cover
```

No need to cover this line - option two would be to extend the corresponding conftest fixture for synthtext and add a entry for one of the blacklisted files which will be skipped at the end